src/chunk.ts

```diff
@@ -22,6 +19,17 @@ const getChunkId = (cacheKey: string, chunk: Chunk) => {
   return `${cacheKey}:${sha256(chunk)}`;
 };
 
+export const purgeChunksAfterResponses = (
+  chunkBucket: Store<Chunk>,
+  chunkIds: string[]
+) => {
+  setTimeout(() => {
+    chunkIds.forEach(chunkId => {
+      chunkBucket.del(chunkId);
```

Comment:
ioredis supports passing an array of keys instead of calling the `del` method repeatedly. It should reduce the amount of redis calls that go over the network.

We can destructure the array into params like this:

```typescript
chunkBucket.del(...chunkIds);
```

It implies that we change the typing of the store from `del: (key: string) => Promise<void>;` to `del: (...key: string[]) => Promise<void>;` and change the `del` method of the in-memory store accordingly.